docs/man_pages/project/testing/emulate-ios.md

```diff
@@ -13,6 +13,7 @@ Before running the iOS Simulator, verify that you have met the following require
 * You have installed Xcode. The version of Xcode must be compatible with the ios-sim-portable npm package on which the NativeScript CLI depends. For more information, visit https://www.npmjs.org/package/ios-sim-portable
 
 Options:
+* `--logging` - If set, prints the standard output of the running application.
```

Comment:
Why is this "standard" output remark? How does the "standard output" differ from the "output".